image-sync/oc-mirror/deploy/image-sync-template.yml

```diff
@@ -0,0 +1,149 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
```

Comment:
`oc` dependency is ok for now but we will need to repackage later since `oc` is not available in EV2